dolphinscheduler-service/pom.xml

```diff
@@ -52,14 +52,8 @@
         </dependency>
 
         <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-quartz</artifactId>
-            <exclusions>
-                <exclusion>
-                    <groupId>org.springframework.boot</groupId>
-                    <artifactId>spring-boot-starter</artifactId>
-                </exclusion>
-            </exclusions>
+            <groupId>com.cronutils</groupId>
+            <artifactId>cron-utils</artifactId>
         </dependency>
         <dependency>
             <groupId>org.quartz-scheduler</groupId>
```

Comment:
Hey I don't have my computer at hands, do we still depend on this module in `-service` module?

Reply:
Right now, we have some class rely on the quartz module to do some cron parsing. We need to refactor the CronUtils, then we can remove the `org.quartz-scheduler ` module. I add a `todo` in `CronUtils`, it will take some to refactor this  util.